src/components/ArgosBanner.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+import argosEmblem from "@site/src/img/argos-emblem.png";
+
+export const ArgosBanner = () => (
```

Comment:
We need an image to put it in readme, it must be easy. We should also provide the markdown code to copy/paste.